spec/features/workflow/schedule_spec.rb

```diff
@@ -66,4 +76,32 @@ def then_i_see_the_edition_has_a_set_scheduled_publishing_date_and_time
     )
     expect(find_field("scheduled[time]").value).to eq("11:00pm")
   end
+
+  def when_i_click_on_schedule
+    click_on "Schedule"
+  end
+
+  def and_i_select_the_content_has_been_reviewed_option
+    choose I18n.t!("schedule.confirmation.review_status.reviewed")
+  end
+
+  def and_click_publish
+    click_on "Publish"
+  end
+
+  def then_i_see_a_confirmation_that_the_edition_has_been_scheduled
+    expect(page).to have_content(I18n.t!("schedule.scheduled.title"))
+  end
+
+  def then_i_see_the_edition_has_been_scheduled
+    expect(page).to have_content(I18n.t!("user_facing_states.scheduled.name"))
+  end
+
+  def and_i_can_no_longer_see_a_schedule_action
+    expect(page).not_to have_link("Schedule")
+  end
+
+  def and_i_can_no_longer_edit_the_content
+    expect(page).not_to have_link("Change")
```

Comment:
I'm not sure if this holds right. As even when the content isn't editable they can change topics and that would mean the page has a "Change" link

Reply:
@kevindew we do use this approach already for the `edition_spec`. The reason it works it because topics are optional for a document type - it was unclear what other visual queue we could use to test this with.

Reply:
Oh yeah shoot I forgot about that being conditional.

Reply:
Are we good to keep this in?